### PR TITLE
Added homework_5_161122 (Tasks 1-8)

### PR DESCRIPTION
**Pull request by kate-lvlup**
Hi Dima,
Hope you are well!
Could you check my homework5?

Thanks in advance!

### CODE REVIEW

File: homework_5_161122/src/LessonFiveTaskEight.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        for (int i = 0; i < height; i++) {
+            for (int j = 0; j < width; j++) {
+                valueArrayThree[i][j] = valueArrayOne[i][j] + valueArrayTwo[i][j];
+            }
+        }
+        for (int i = 0; i < height; i++) {
+            for (int j = 0; j < width; j++) {
+                System.out.print(valueArrayThree[i][j] + ", ");
+            }
+            System.out.println();
```
Comment (YeaaaaH) [inline homework_5_161122/src/LessonFiveTaskEight.java:62]: you can simplify it, use previous loop to print value

File: homework_5_161122/src/LessonFiveTaskOne.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        int valueArrayLength = valueScanner.nextInt();
+        System.out.println("Enter the elements of the new array");
+        int[] valueInputArray = new int[valueArrayLength];
+        for (int i = 0; i < valueArrayLength; i++) {
+            valueInputArray[i] = valueScanner.nextInt();
+        }
+        int valueSum = 0;
+        for (int i = 0; i < valueArrayLength; i++) {
+            valueSum += valueInputArray[i];
+        }
```
Comment (YeaaaaH) [inline homework_5_161122/src/LessonFiveTaskOne.java:19]: this one also can be simplified, use previous loop

File: homework_5_161122/src/LessonFiveTaskThree.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            System.out.println("Array length: " + 0);
+        } else {
+            for (int i = 0; i < valueArray.length; i++) {
+                valueArray[i] = Math.random() * 20 - 10;
+                System.out.println("Array element [" + i + "]: " + valueArray[i]);
+            }
+            double valueSum = 0;
+            for (int i = 0; i < valueArray.length; i++) {
+                valueSum += valueArray[i];
+            }
```
Comment (YeaaaaH) [inline homework_5_161122/src/LessonFiveTaskThree.java:18]: use previous loop to calculate

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
need to fix comments

File: homework_5_161122/src/LessonFiveTaskEight.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_5_161122/src/LessonFiveTaskEight.java:62]: Fixed loop "Sum of arrays"

File: homework_5_161122/src/LessonFiveTaskThree.java (hunk 3 above)
Comment (kate-lvlup) [reply homework_5_161122/src/LessonFiveTaskThree.java:18]: Fixed. Added "valueSum" variable to previous loop.

File: homework_5_161122/src/LessonFiveTaskOne.java (hunk 2 above)
Comment (kate-lvlup) [reply homework_5_161122/src/LessonFiveTaskOne.java:19]: Fixed. Added variable "valueSum" to previous loop.

**Comment by kate-lvlup** [comment]
All issues were fixed and pushed to branch "homework_5_161122" on GitHub

**Review by YeaaaaH — APPROVED** [review]

**Comment by kate-lvlup** [comment]
All commits were merged into master branch.

**Pull request merged**